package.json

```diff
@@ -152,6 +152,7 @@
     "karma-spec-reporter": "0.0.32",
     "karma-verbose-reporter": "0.0.6",
     "karma-viewport": "^1.0.6",
+    "lodash": "^4.17.19",
```

Comment:
do we need to add it here? Isn't it sufficient to have lodash bumped in `package-lock.json`?

Reply:
I think it is better to keep it this way e.g. in case one wants to regenerate the lock as well the for the yarn upgrade.

Reply:
I just don't want us to be in the habit of including sub-dependencies in `package.json`, is this one special in some way? Are we waiting for something that will let us remove this?

Reply:
Until all those sub-dependencies switch to force using newer version, I think we need to control the version at root.
We have few other dev-dependencies e.g. `acorn`, `static-eval` that controlled like this.

Reply:
OK, fair enough.